### ISSUE TITLE
document support multiple version

### ISSUE THREAD

**Issue by Anilople** [apolloconfig/apollo#4327]
**Is your feature request related to a problem? Please describe.**
When user view the document https://www.apolloconfig.com/ , user may be only care about the document match their apollo's version.

For example, when user user's apollo version is 1.5.0, he/she may be confuse when see the feature above 1.5.0

**Describe the solution you'd like**
When someone use elasticsearch, its official site https://www.elastic.co/guide/en/elasticsearch/reference/7.17/index.html support choose the matched version of document.
![image](https://user-images.githubusercontent.com/15523186/165523338-6770a373-63a0-460d-9ff1-e062ebada285.png)

Maybe we can add a button `version` on the top right of site?
![image](https://user-images.githubusercontent.com/15523186/165523994-5621cf28-28e0-4aef-a469-3826247983c3.png)
and jump to that version's document which mark by the git tag.

**Additional context**
It's will improve the experience about the document, decrease the user's confuse.

**Comment by nobodyiam** [comment]
Good idea! I suppose this will also make the documentation more precise and simple, e.g. we don't need to mark this section is for 2.0.0+, this section is for 1.9.0+, etc.
